tests/test_cluster/test_elbow.py

```diff
@@ -69,6 +70,10 @@
 
 y = np.array([0, 0, 1, 1, 0, 0, 1, 1, 0, 1, 0, 0, 1, 1, 1, 1, 0, 0])
```

Comment:
While we're here ... would you mind changing this to a pytest.fixture and use `make_blobs` to create the dataset? Similar to what's happening in test_icdm.py?

Reply:
Sure. I'll do that.